.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,7 +10,7 @@ assignees: ''
 > Note: This repo is for the Python devguide. If you are requesting an
 enhancementfor the Python language or CPython interpreter,
 then the CPython issue tracker is better
-suited for this report: https://bugs.python.org
+suited for this report: https://devguide.python.org/tracker/
```

Comment:
I think it makes sense to link directly to https://github.com/python/cpython/issues here

Reply:
Updated x 3!

Reply:
Thanks!